assets/stylesheets/gu-sass/layout.scss

```diff
@@ -65,21 +64,27 @@ $gu-v-spacing: 12px;
   }
 }
 
-// The element into which React content is injected.
 .gu-content-wrapper {
   height: 100%;
 
   > div {
-    display: flex;
     height: 100%;
-    flex-direction: column;
-  }
-
-  header {
-    flex: 0 0 auto;
   }
 
-  section {
+  .gu-content {
+    @supports (display: flex) {
+      display: flex;
+    }
+    // height: 100%;
+    flex-direction: column;
     flex: 1 0 auto;
+
+    .gu-content-margin {
+      flex: 1 0 auto;
+    }
+
+    header, footer {
+      flex: 0 0 auto;
+    }
```

Comment:
Ought all of this `flex` stuff go inside the `@supports` block? I know older browsers will ignore it, but it would keep things clearer for developers